net-print/fax4cups/fax4cups-1.29-r1.ebuild

```diff
@@ -24,14 +25,13 @@ RDEPEND="${DEPEND}
 	app-admin/sudo
 "
 
-S=${WORKDIR}/fax4CUPS-${PV}
-
 src_install() {
 	doman fax4CUPS.1
 
 	exeinto $(cups-config --serverbin)/backend
 	insinto /usr/share/cups/model
 
+	local i
 	for i in hylafax efax mgetty-fax; do
 		if use $i
 		then
```

Comment:
(Can't comment on correct line, but the postinst should say ${EROOT}/etc/init.d/cups restart, even though it's unlikely to matter...)

(Also presumably this is wrong for systemd)